content/en/docs/architecture/images.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "OCP Builder Images"
+date: 2020-01-22T23:27:39-04:00
+draft: false
+---
+
+Each component of OpenShift Container Platform is tested and released as a container image. When a pull request for the component repo is created, a CI job builds the image for the component based on the PR. We call this type of images _CI images_. When the OCP is released as a product, the image of each component is built by ART team. We call those images _ART images_. In this document, we illustrate the build process of those images.
+
+## CI Images
+
+CI images are built according to the ci-operator's configuration, as described in [building-container-images](/docs/architecture/ci-operator/#building-container-images). Take `cluster-etcd-operator` (branch `release-4.6`) for an example:
+
+```yaml
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+images:
+- dockerfile_path: Dockerfile.rhel7
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6
+  to: cluster-etcd-operator
+```
+
+where `Dockerfile.rhel7` in the repo looks like
+
+```Dockerfile
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 AS builder
+#skip other instructions
+RUN make build --warn-undefined-variables
+
+FROM registry.ci.openshift.org/ocp/4.6:base
+COPY --from=builder /go/src/github.com/openshift/cluster-etcd-operator/bindata/bootkube/bootstrap-manifests /usr/share/bootkube/manifests/bootstrap-manifests/
+#skip other instructions
+```
+
+When the CI image is built, the images in `From` directives of `Dockerfile.rhel7` are replaced according to the `ci-operator`'s configuration above:
+* `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6` is replaced by `ocp_builder_rhel-8-golang-1.15-openshift-4.6`
+* `registry.ci.openshift.org/ocp/4.6:base` is replaced by `base`
+
+where `ocp_builder_rhel-8-golang-1.15-openshift-4.6` and `base` are defined in `base_images`:
```

Comment:
Delete this section - and we do not really care to explain mirroring and cluster-local images here. It's not important to this topic IMHO - a user should not worry about it.